worldeditadditions_core/register/check.lua

```diff
@@ -0,0 +1,14 @@
+function worldeditadditions_core.register_command(def)
```

Comment:
Might be worth documenting what this function does. Also, the filename doesn't really match the function at all?

Reply:
This is going to be refactored. `wea.core` stuff shouldn't actually be in this pull request (it's still very WIP) but I was storing it on the branch so 🤷.